Yafc/Windows/MainScreen.cs

```diff
@@ -365,8 +371,8 @@ private void SettingsDropdown(ImGui gui) {
         if (gui.BuildContextMenuButton("Undo", "Ctrl+" + ImGuiUtils.ScanToString(SDL.SDL_Scancode.SDL_SCANCODE_Z)) && gui.CloseDropdown()) {
             project.undo.PerformUndo();
         }
-
-        if (gui.BuildContextMenuButton("Save", "Ctrl+" + ImGuiUtils.ScanToString(SDL.SDL_Scancode.SDL_SCANCODE_S)) && gui.CloseDropdown()) {
+        
```

Comment:
You added some whitespace (formatter should remove this)

Reply:
I see that the [Debug build check / build (Debug)](https://github.com/shpaass/yafc-ce/actions/runs/13303069033/job/37148936222?pr=412) pipeline also is complaining about formatting